### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by Adanzeakonobi**
In this branch  added the following; 
- index.html that contains body part
- style.css that contains position and colour
- .gitignore
- image folders

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,57 +1,213 @@
...
+              <img class="image-correction" src="/icons/medium.png" alt="Medium"/>
+            </a>
+          </li>
+        </ul>
+    </section>
+  </header>
+   <main>
+      <section class="new-container">
+        <div class="work-container">
+          <h1 class="recent-work">My Recent Works</h1>
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: index.html [hunk 2]
```diff
@@ -1,57 +1,213 @@
...
+          <hr class="about-hr">
+        </div>
+        <div class="image-p">
+          <img class="image-placeholder" src="Skeleton/placeholder.png" alt="place holder">
+          <div class="multi">
+           <h1 class="multi-post">Multi-Post Stories</h1>
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: index.html [hunk 3]
```diff
@@ -1,57 +1,213 @@
...
+            <li class="list">html</li>
+            <li class="list">bootstrap</li>
+            <li class="list">Ruby</li>
+           </ul>
+           <button class="project" type="button">See Project</button>
+          </div>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h1 class="art-print">Profesional Art Printing Data</h1>
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: index.html [hunk 4]
```diff
@@ -1,57 +1,213 @@
...
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h1 class="art-print">Profesional Art Printing Data</h1>
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: index.html [hunk 5]
```diff
@@ -1,57 +1,213 @@
...
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: index.html [hunk 6]
```diff
@@ -1,57 +1,213 @@
...
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: index.html [hunk 7]
```diff
@@ -1,57 +1,213 @@
...
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: index.html [hunk 8]
```diff
@@ -1,57 +1,213 @@
...
```
Comment (phelian23) [inline index.html]: - [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).

File: style.css [hunk 9]
```diff
@@ -1,101 +1,394 @@
...
+.about-hr {
+  border: 1px dotted #000;
+  margin-bottom: 15px;
+}
+
+.image-p {
+  display: grid;
+  padding: 20px;
+  margin: 20px;
+}
```
Comment (phelian23) [inline style.css]: - [ ] Kindly make sure that your design match the Figma template design as much as possible (e.g. font, colors, images, tests, margins)
Your design
Capture
Capture2
Capture3

Figma template
figma2
figma3
figma

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Adanzeakonobi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
- All linter checks are passing ✔️
- Descriptive commits and correct use of the Git Flow 🔥
- Descriptive PR ✔️
- Professional README ✔️
- Clean code ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -1,57 +1,214 @@
...
+            <p class="daily-selection">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
```
Comment (AnselemOdims) [inline index.html:91]: - [ ] As already pointed out by the previous reviewer, kindly make sure the size of your button design matches the original design size. In this case, the height is smaller compared to the original design

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Adanzeakonobi, 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AmaduKamara — APPROVED** [review]
Hi @Adanzeakonobi, 

Good job implementing the changes asked by the previous reviewers. Keep doing great 👍
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**